platforms/documentation/docs/src/docs/userguide/dep-man/dependency-resolution/dependency_resolution_basics.adoc

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2024 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[dependency_resolution_basics]]
+= Dependency Resolution Basics
+
+Dependency resolution in Gradle can largely be thought of as a two-step process.
+
+First, the _graph resolution_ phase constructs the dependency graph based on declared dependencies.
+Second, the _artifact resolution_ phase fetches the actual files (artifacts) for the resolved components:
+
+1. **Graph resolution phase**:
+- Driven by declared dependencies and their metadata
+- Uses attributes defined in the configuration
+
+2. **Artifact resolution phase**:
+- Based on nodes in the resolved dependency graph
+- Matches the variant of each node
```

Comment:
What does `Matches the variant of each node` mean here? I think this wording can be improved.

Reply:
Updated the wording.